chore(agw): Clean up unused imports

voisey opened the pull request:
## Summary

Removes a selection of unused imports found via `cd $MAGMA_ROOT/lte/gateway/python && make check` in the magma-dev VM. Also removes the corresponding dependencies from BUILD.bazel files when they've been used.

## Test Plan

- [x] magma-dev: `cd lte/gateway && make test_python_service UT_PATH=python/magma/mobilityd/tests/`
- [x] magma-dev: `cd lte/gateway && make test_python_service UT_PATH=python/magma/pipelined/tests/`
- [x] magma-dev: `bazel test --cache_test_results=no lte/gateway/python/magma/mobilityd/tests:ip_alloc_dhcp_test`
- [x] magma-dev: `bazel test --cache_test_results=no lte/gateway/python/magma/pipelined/tests:test_rpc_servicer`
- [x] magma-test: Run modified integration tests, i.e. `cd lte/gateway/python/integ_tests && make integ_test TESTS=s1aptests/<test>.py`
- [x] `cd magma/lte/gateway && fab federated_integ_test:build_all=True`

voisey committed:
chore(agw): Clean up unused imports

Signed-off-by: Cameron Voisey <cameron.voisey@tngtech.com>

themarwhal reviewed (approved):
Lgtm! Were there no fixups under the orchestrator directory?

uri200 reviewed (changes requested):
Please since this touches subcriberd_db_client, rebase to latest and run [s1ap federated integ test ](https://docs.magmacore.org/docs/next/feg/s1ap_federated_tests#s1ap-federated-integration-tests).

```
cd magma/lte/gateway
fab federated_integ_test:build_all=True

```

LKreutzer reviewed (approved):
Bazel changes lgtm - thanks for the cleanup Cameron!

ardzoht reviewed (approved):
integtests changes lgtm

voisey committed:
chore(agw): Clean up more unused imports

Signed-off-by: Cameron Voisey <cameron.voisey@tngtech.com>

voisey commented:
Thanks for the reviews! I've pushed a second commit removing a few more imports and reran the tests mentioned in the summary. Are there any more I should run?

@themarwhal Good point! I found one case in the orc8r, which I removed in the recent commit.

@uri200 I rebased and ran the federated integ test, which works :slightly_smiling_face:
